src/main/java/org/springframework/data/redis/connection/RedisSetCommands.java

```diff
@@ -153,6 +154,16 @@ public interface RedisSetCommands {
 	 */
 	Long sInterStore(byte @NonNull [] destKey, byte @NonNull [] @NonNull... keys);
 
+	/**
+	 * Returns the cardinality of the set which would result from the intersection of all the given sets.
+	 *
+	 * @param keys must not be {@literal null}.
+	 * @return {@literal null} when used in pipeline / transaction.
+	 * @see <a href="https://redis.io/commands/sintercard">Redis Documentation: SINTERCARD</a>
+	 * @since 3.4
```

Comment:
I believe the `@since` should be 4.0 now (quite a bit of time has passed since initially submitted).